docs/it/token_endpoint.rst

```diff
@@ -186,40 +174,32 @@ L’ID Token è un JSON Web Token (JWT) che contiene informazioni sull’utente
 
 
 .. list-table:: 
-   :widths: 20 20 40 20
+   :widths: 20 60 20
    :header-rows: 1
 
    * - **Claim**
-     - **Tipo**
      - **Descrizione**
      - **Obbligatorio**
    * - **iss** 
-     - 
      - Identificatore dell’OP che lo contraddistingue univocamente nella federazione nel formato Uniform Resource Locator (URL). Il client è tenuto a verificare che questo valore corrisponda all’OP chiamato.
      - 
    * - **sub** 
-     - 
      - Per il valore di questo parametro fare riferimento allo standard “OpenID Connect Core 1.0”, “Pairwise Identifier Algorithm”. 
-     -
-   * - **aud** 
      - 
+   * - **aud** 
      - Contiene il client ID. Il client è tenuto a verificare che questo valore corrisponda al proprio client ID.
      - 
    * - **acr** 
-     - 
      - Livello di autenticazione effettivo. Può essere uguale o superiore a quello richiesto dal client nella Authentication Request.
      - 
    * - **at_hash** 
-     - 
      - Hash dell’Access Token. Il suo valore è la codifica base64url della prima metà dell’hash del valore access_token, usando l’algoritmo di hashing indicato in **alg** nell’header dell’ID Token. Il client è tenuto a verificare che questo valore corrisponda all’*access token* restituito insieme all’ID Token.
      - 
    * - **iat** 
-     - UNIX Timestamp
-     - Data/ora di emissione del token in formato NumericDate, come indicato in :rfc:`7519#JWT`.
+     - UNIX Timestamp. Data/ora di emissione del token in formato NumericDate, come indicato in :rfc:`7519#JWT`.
```

Comment:
- UNIX Timestamp con la data di emissione del JWT, codificato come NumericDate come indicato in
       `RFC7519 - JSON Web Token (JWT) <https://www.rfc-editor.org/rfc/rfc7519.html>`_